var/spack/repos/builtin/packages/py-psyclone/package.py

```diff
@@ -34,12 +36,10 @@ class PyPsyclone(PythonPackage):
     version("2.0.0", sha256="94766ffda760404af99f85d70341376192e4a1b8e16e7ae5df980038898a9c41")
     version("1.5.1", sha256="f053ad7316623b2a4002afc79607abda3b22306645e86f2312d9f3fe56d312dc")
 
-    depends_on("fortran", type="build")  # generated
-
     # Current dependencies
```

Comment:
According to `requirements.txt` for the new version, the requirements are:

```
termcolor
graphviz
```

They are also listed in `setup.py`.

So `py-termcolor` and `py-graphviz` are also required for the new version though I'm not seeing either listed here.

Reply:
Oops, added. I've not used `when` for these as they've been optional dependencies for a long time.